CHANGELOG.develop

```diff
@@ -1788,6 +1788,7 @@ Other:
 **** PHP
 - Added company-php (thanks to jim and Eivind Fonn)
 - Fixed php-company autocompletion (thanks to Dela Anthonio)
+- Updated documentation regarding autocompletion (thanks to Serghei Iakovlev)
```

Comment:
Instead of adding a line related to documentation here, please add your name in the `Various documentation improvements` list (at the end of the file). This is to shorten the changelog a bit. All documentation changes were recorded there.

Reply:
@sdwolfz However, the build failed because someone set up tests in this way. That is why I added this entry here.

Reply:
Are you referring to `ci/circleci: CHANGELOG.develop updated (optional)`? while it shows it's failed it does not actually block merging.